unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -558,21 +543,10 @@ public bool IsEntityNFT()
 
     bool IsEntityAFloor()
     {
-        if (rootEntity.meshesInfo?.currentShape == null)
-            return false;
-        if (rootEntity.meshesInfo.renderers?.Length <= 0)
-            return false;
-        if (rootEntity.meshesInfo.mergedBounds.size.y >= 0.02f)
-            return false;
-        if (rootEntity.gameObject.transform.position.y >= 0.05f)
-            return false;
+        if (GetCatalogItemAssociated()?.category == BuilderInWorldSettings.FLOOR_CATEGORY)
+            return true;
```

Comment:
this could be a one liner right?
`return GetCatalogItemAssociated()?.category == BuilderInWorldSettings.FLOOR_CATEGORY;`